controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -118,7 +118,7 @@ http {
     {{ if $cfg.DisableAccessLog }}
     access_log off;
     {{ else }}
-    access_log /var/log/nginx/access.log upstreaminfo if=$loggable;
+    access_log {{ $cfg.AccessLogPath }} upstreaminfo if=$loggable;
     {{ end }}
     error_log  /var/log/nginx/error.log {{ $cfg.ErrorLogLevel }};
```

Comment:
Why not also add a variable for error_log?

Reply:
I just made it configurable in the PR since I hadn't any reason not to add it